docs/src/pages/components/rating/react.mdx

```diff
@@ -12,224 +27,225 @@ Rating shows users a star rating of a product or service (currently read-only).
+    /* OR */
+    .amplify-rating-icon-filled {
+      color: var(--amplify-colors-purple-60);
+    }
+
+    .amplify-rating-icon-empty {
+      color: var(--amplify-colors-blue-60);
+    }
+    ```
+
+  </ExampleCode>
 </Example>
 
-#### Sizes
-
-The size variations on the Rating component can be overridden at the individual component level
-
-```css
-/* styles.css */
-.large-rating[data-size='small'] {
-  font-size: 40px;
-  line-height: 40px;
-}
-```
-
-```jsx
-import './styles.css';
+### Sizes
 
-<Rating value={2.5} className="large-rating" size="small" />;
-```
+The size variations on the Rating component can be overridden at the application level using CSS variables
 
 <Example>
-  <Rating value={2.5} className="large-rating" size="small" />
+  <RatingGlobalSizeExample />
+  <ExampleCode>
+  ```css
+    /* styles.css */
+    :root {
+      --amplify-components-rating-small-size: var(--amplify-font-sizes-xxxl);
+    }
+    /* OR */
+    .amplify-rating[data-size=small] {
+      font-size: var(--amplify-font-sizes-xxxl);
+      line-height: var(--amplify-font-sizes-xxxl);
+    }
+  ```
+
+  </ExampleCode>
 </Example>
 
-### Application wide styling
-
-All Rating components within an application can be styled using CSS variables at the root level
-
-#### Filled and empty icon colors
-
-The empty and filled icon colors can be overridden using css variables
+### Local styling
 
-```css
-:root {
-  --amplify-components-rating-filled-color: purple;
-  --amplify-components-rating-empty-color: black;
-}
-```
+The Rating component styling props can be overridden using a custom class name and the built-in CSS classes.
 
-```jsx
-import { Rating } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+### Filled and empty icon colors
 
-<Rating value={2.5} className="application-color-override" />;
-```
+The empty and filled icon colors can be overridden using a custom CSS class on your Rating component and with the `.amplify-rating-icon-filled` and `.amplify-rating-icon-empty` classes
 
 <Example>
-  <Rating value={2.5} className="application-color-override" />
-</Example>
+  <RatingStyleColorExample />
+  <ExampleCode>
+    ```css
+    .my-rating-component .amplify-rating-icon-filled {
+      color: var(--amplify-colors-purple-60);
+    }
 
-### Sizes
+    .my-rating-component .amplify-rating-icon-empty {
+      color: var(--amplify-colors-blue-60);
+    }
+    ```
 
-The size variations on the Rating component can be overridden at the application level using CSS variables
+  </ExampleCode>
+  <ExampleCode>
+    ```jsx file=./examples/RatingStyleColorExample.tsx
+
+    ```
 
-```css
-:root {
-  --amplify-components-rating-small-size: 40px;
-}
-```
+  </ExampleCode>
+</Example>
 
-```jsx
-import { Rating } from '@aws-amplify/ui-react';
-import '@aws-amplify/ui-react/styles.css';
+#### Sizes
 
-<Rating value={2.5} size="small" />;
-```
+The size variations on the Rating component can be overridden at the individual component level
```

Comment:
The Rating component in this example displays as `xxxl`. Is this expected, or should it be `small` to match the prop passed in at the component level?